back/src/bsvhu/validation/schema.ts

```diff
@@ -88,6 +88,10 @@ const rawBsvhuSchema = z.object({
     .boolean()
     .nullish()
     .transform(v => Boolean(v)),
+  emitterNotOnTD: z.coerce
+    .boolean()
+    .nullish()
+    .transform(v => Boolean(v)),
```

Comment:
Ca suffit ca non ? A priori un coerce est jamais nullable et sera toujours un `Boolean`
```suggestion
  emitterNotOnTD: z.coerce .boolean(),
```